### DISCUSSION TITLE
Multiple nuxt projects on one server. They use the wrong env

### DISCUSSION THREAD

**Discussion by adnseek** [nuxt/nuxt#21432] [Questions]
### Environment

The issue is. i have 2 domains running on 1 server, both nuxt 3, same codebase, different .env.

deployment with pm2.

1) different nginx configuration - working
2) different redis db id - working

its all working, but both pages seem to use the same .env values. and i cannot find out why.

Do they overwrite each other ?

Cheers and thanks
Adrian

### Reproduction

already described

### Describe the bug

2 domains - different env - but both domains use the same .env values - do they overwrite each other ?

**Comment by danielroe** [comment]
I think this is likely a server issue. How are you setting your environment variables?

**Reply by adnseek** [reply]
the envs are inside of each domain subfolder like /home/forge/domain1/.env /home/forge/domain2/.env

**Reply by adnseek** [reply]
they are different but both pages show the contents of domain1/.env

**Reply by danielroe** [reply]
Nuxt doesn't read `.env` into the environment when running in production. You need to set the variables yourself in a starter script or in some other way.

**Comment by arinzejustin** [comment]
Can please someone help me with nuxt server side authentication and please I want to use cookie for authenication then assign isAuthenticated a boolean based on return value for my middleware auth
